changelogs/fragments/382-zabbix_user.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - zabbix_user - removed some default values because a configuration should be changed only if specified a parameter (https://github.com/ansible-collections/community.zabbix/pull/382).
```

Comment:
```suggestion
  - zabbix_user - removed some of the default values because a configuration should be changed only if specified as a parameter (https://github.com/ansible-collections/community.zabbix/pull/382).
```